lib/pb.ml

```diff
@@ -11,7 +11,7 @@ open Unsigned
 
 exception Parse_error
 
-module Wire_type = Pb_wire_type
+module Wire_type = Wire_type
```

Comment:
Is this line necessary at all after the change?

Reply:
I think it is. Otherwise the `Wire_type` in the .mli will have no implementation.